packages/scene-composer/package.json

```diff
@@ -46,7 +46,7 @@
     "convert-svg": "npx @svgr/cli --out-dir src/assets/auto-gen/icons/ --typescript --index-template tools/index-template.js -- src/assets/icons/",
     "release": "run-s compile copy-assets",
     "copy-assets": "copyfiles -e \"**/*.tsx\" -e \"**/*.ts\" -e \"**/*.snap\" -e \"**/*.js\" -e \"**/*.jsx\" -e \"**/*.json\" \"src/**/*\" dist/",
-    "lint": "eslint . --max-warnings=485",
+    "lint": "eslint . --max-warnings=712",
```

Comment:
Isn't this supposed to decrease the number of warnings? Why are we increasing the limit here?

Reply:
We are adding a new linter that is increasing the warnings as we now include warnings from the increased coverage. While this might appear bad, it's actually really good to help us improve our code.

Reply:
It's not supposed to decrease the number of warnings, it's enforcing better practices in the code through increasing warnings. imo we should fix those, the point of adding the linter logic was to find other areas we may have issues.